frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -461,7 +461,7 @@ describe("scenarios > collection_defaults", () => {
       cy.findByText("Orders");
     });
 
-    it.skip("collections without sub-collections shouldn't have chevron icon (metabase#14753)", () => {
```

Comment:
Please never delete the reference to the issue. We need that for the future reference to easier spot the regressions.

Just unskip the test.
Thanks